.build/cassandra-deps-template.xml

```diff
@@ -244,8 +244,8 @@
     <!-- end of chronicle-queue -->
 
     <dependency>
-      <groupId>org.fusesource</groupId>
-      <artifactId>sigar</artifactId>
+      <groupId>com.github.oshi</groupId>
+      <artifactId>oshi-core</artifactId>
```

Comment:
this also fetches this into the project:

[resolver:resolve] Downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-api/2.0.9/slf4j-api-2.0.9.pom
[resolver:resolve] Downloaded https://repo1.maven.org/maven2/org/slf4j/slf4j-api/2.0.9/slf4j-api-2.0.9.pom (3 KB at 152.8 KB/sec)
[resolver:resolve] Downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-parent/2.0.9/slf4j-parent-2.0.9.pom
[resolver:resolve] Downloaded https://repo1.maven.org/maven2/org/slf4j/slf4j-parent/2.0.9/slf4j-parent-2.0.9.pom (16 KB at 947.1 KB/sec)
[resolver:resolve] Downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-bom/2.0.9/slf4j-bom-2.0.9.pom
[resolver:resolve] Downloaded https://repo1.maven.org/maven2/org/slf4j/slf4j-bom/2.0.9/slf4j-bom-2.0.9.pom (5 KB at 300.3 KB/sec)

I am not sure we should depend on this. Maybe we should exclude it?